tests/kafkatest/services/kafka/kafka.py

```diff
@@ -842,8 +842,7 @@ def start_node(self, node, timeout_sec=60, **kwargs):
                     self.close_port(controller_listener)
 
         self.security_config.setup_node(node)
-        if self.quorum_info.using_zk or self.quorum_info.has_brokers: # TODO: SCRAM currently unsupported for controller quorum
-            self.maybe_setup_broker_scram_credentials(node)
+        self.maybe_setup_broker_scram_credentials(node)
```

Comment:
this method will call `maybe_setup_broker_scram_credentials` to use zk connection. Is it safe if the e2e is under kraft?